aspnetcore/signalr/publish-to-azure-web-app.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Publish an ASP.NET Core SignalR app to Azure Web App
+author: rachelappel 
+description: Publish an ASP.NET Core SignalR app to Azure Web App
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 04/20/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/publish-to-azure-web-app
+---
+
+# Publish an ASP.NET Core SignalR app to an Azure Web App
+
+[Azure Web App](https://docs.microsoft.com/azure/app-service/app-service-web-overview) is a [Microsoft cloud computing](https://azure.microsoft.com/) platform service for hosting web apps, including ASP.NET Core.
+
+## Publish the app
+
+Visual Studio provides built-in tools for publishing to an Azure Web App. Visual Studio Code user can use [Azure CLI](https://docs.microsoft.com/cli/azure) commands to publish apps to Azure. This article covers publishing using  the tools in Visual Studio. To publish an app using Azure CLI, see [Publish an ASP.NET Core app to Azure with command line tools](xref:tutorials/publish-to-azure-webapp-using-cli).
+
+Right-click on the project in **Solution Explorer** and select **Publish**. Confirm that **Create new** is checked in the **Pick a publish target** dialog, and select **Publish**.
+
+![Pick publish target](publish-to-azure-web-app/_static/pick-publish-target-dialog.png)
+
+Enter the following information in the **Create App Service** dialog and select **Create**.
+
+| Item | Description |
+| ---- | ----------- |
+| **App name** | A unique name of the app. |
+| **Subscription** | The Azure subscription that the app uses. |
+| **Resource Group** | The group of related resources to which the app belongs.  |
+| **Hosting Plan** | The pricing plan for the web app. |
+
+![Create app service](publish-to-azure-web-app/_static/create-app-service-dialog.png)
+
+Visual Studio completes the following tasks:
+
+* Creates a Publish Profile containing publish settings.
+* Creates or uses an existing *Azure Web App* with the provided details.
+* Publishes the app.
+* Launches a browser, with the published web app loaded.
+
+Notice the format of the URL for the app is *{app name}.azurewebsites.net*. For example, an app named `SignalRChattR` has a URL that looks like *https://signalrchattr.azurewebsites.net*.
+
+![Appname-azurewebsites-net](publish-to-azure-web-app/_static/app-azurewebsites-net.png)
+
+> [!NOTE]
+> If an HTTP 502.2 error occurs, it is because some ASP.NET Core components haven't been loaded. To resolve this error, complete the steps later in the *Add ASP.NET Core Runtime extensions* section of this article.
```

Comment:
Not italic, a link (generally when referring to a section in the article).
`[Add ASP.NET Core Runtime extensions](#runtime-extensions)`